it/google-cloud-platform/src/main/java/org/apache/beam/it/gcp/LoadTestBase.java

```diff
@@ -215,19 +219,23 @@ private void computeDataflowMetrics(
     // cost info
     LOG.info("Calculating approximate cost for {} under {}", launchInfo.jobId(), project);
     TimeInterval workerTimeInterval = getWorkerTimeInterval(launchInfo);
-    metrics.put(
-        "RunTime",
-        (double)
-            Timestamps.between(workerTimeInterval.getStartTime(), workerTimeInterval.getEndTime())
-                .getSeconds());
+    if (workerTimeInterval.hasStartTime() && workerTimeInterval.hasEndTime()) {
```

Comment:
Why do we need this change? Is it just copying Beam? Or something else? If we're copying Beam, can we just use Beam's implementation?

Reply:
Same question below